tests/testthat/test-get_acoustic_detections.R

```diff
@@ -167,22 +177,25 @@ test_that("get_acoustic_detections() allows selecting on acoustic_project_code",
   # Select single value
   single_select <- "demer"
   single_select_df <- get_acoustic_detections(con, acoustic_project_code = single_select)
-  expect_equal(
+  expect_identical(
     single_select_df %>% distinct(acoustic_project_code) %>% pull(),
     c(single_select)
   )
   expect_gt(nrow(single_select_df), 0)
 
   # Selection is case insensitive
-  expect_equal(
-    get_acoustic_detections(con, acoustic_project_code = "demer", limit = TRUE),
```

Comment:
These two (equivalent) queries each create a 33MB object, we should probably look for `acoustic_project_code` values that result in smaller objects for testing, that would speed things up.

Reply:
Try 2015_HOMARUS